### ISSUE TITLE
More precise locale, especially for dates

### ISSUE THREAD

**Issue by Alex131089** [duplicati/duplicati#3150]
- [x] I have searched open and closed issues for duplicates.

----------------------------------------

## Environment info
- **Duplicati version**: 2.0.3.1_experimental_2018-03-16
- **Operating system**: Win10 1709 x64

## Description
While my main language is French, I use some softwares in English, especially to avoid any translation ambiguity. Duplicati let me set the language, but the English that comes with it also comes with the non-sense mm/dd/yyyy date format.
Could it be possible to specify locales more precisely, allowing en-GB (which use dd/mm/yyyy, at least on Linux) for example ?

**Comment by piegamesde** [comment]
Most systems differentiate between the actual language and the formats, which can differ. Duplicati should be using the system settings if available.

**Comment by OscarVanL** [comment]
+1 to this, I'm from the UK (My Windows language setting is 'English (United Kingdom)') and it displays the American date format.

**Comment by sky4055** [comment]
+1, same issue.

**Comment by vic-t** [comment]
Yes, please allow for a differentiation of language and other locale settings such as formatting for numbers, date and time.

**Comment by gsharratt** [comment]
It would be great if Duplicati simply used the ISO date and time format everywhere: YYYY-MM-DD HH:MM:SS
(Edited 2026-08-08 to change the semi-colons to colons.)

**Comment by dinfinity** [comment]
+1. ISO8601 should really be the default for every even remotely technical application (it's an international standard for a reason). Localization and relative time should be addons after that.

**Comment by Tighbu** [comment]
Yes, this is ridiculous. Agree with @dinfinity that it should be ISO8601 format, especially for any logging. The chosen display format may change over time but logs are long-term historical documents so the format should be consistent there.
